recipes/ckanweb-deploy.rb

```diff
@@ -275,6 +275,20 @@
 	mode "0744"
 end
 
+cookbook_file "/etc/supervisor/conf.d/supervisor-ckan-harvest-gather.conf" do
```

Comment:
is this ok to run on N+1 nodes? same for fetch.

Reply:
It shouldn't need to run on N+1 nodes - is there a way to restrict it to a single node?

Reply:
If we don't run it on every server, then we can't freely fail over from one server to another. I would like to run it everywhere if possible.